orb-ui/src/sound/capture.rs

```diff
@@ -24,8 +24,9 @@ impl Default for CaptureLoopSound {
 }
 
 impl CaptureLoopSound {
-    /// Come back to part A when out of range detected
-    pub fn out_of_range(&mut self) {
+    /// Come back to part A of the current playing loop sound
+    /// (when out of range detected)
```

Comment:
```suggestion
```
This function unconditionally restarts the current loop. There is no check for out of range from what I could see. Can you confirm?